src/scripts/profile-setup.ts

```diff
@@ -69,21 +70,21 @@ export async function profileSetup(
       ? await web3Interface.archaeologistFacet.updateArchaeologist(
           fullPeerString,
           diggingFeePerSecond,
-          rewrapInterval,
+          rewrapInterval!,
           freeBondDeposit,
-          maxResTime
+          maxResTime!
         )
       : await web3Interface.archaeologistFacet.registerArchaeologist(
           fullPeerString,
           diggingFeePerSecond,
-          rewrapInterval,
+          rewrapInterval!,
           freeBondDeposit,
-          maxResTime
+          maxResTime!
         );
 
     archLogger.notice(`${txType} Archaeologist`);
     archLogger.info("Please wait for TX to confirm");
-    await tx.wait();
+    await tx?.wait();
```

Comment:
Should we throw an error if tx is not defined? Or if that would be thrown during update/register process, maybe we could make this `tx!` to indicate that if execution gets here that the TX will be defined.